test.sh

```diff
@@ -9,6 +9,7 @@
 set -e
 
 DIR="$(dirname "$0")"
+mkdir -p 'test-results'
```

Comment:
Let's do this in code before we write results like for all the other paths (we create directories for builds and dist).